build.gradle

```diff
@@ -13,38 +12,49 @@ sourceCompatibility = JavaVersion.VERSION_11  // source-code version and must be
 targetCompatibility = JavaVersion.VERSION_11 // bytecode target version
 
 repositories {
-    mavenCentral()
+    jcenter()
+    maven { url 'https://jitpack.io' }
 }
 
+def junit5Version = '5.6.0'
+def zalandoVersion = '0.23.0'
+
 dependencies {
+    implementation group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: 'v0.3.0'
+
     implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.10'
     implementation group: 'commons-io', name: 'commons-io', version: '2.6'
     implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.9'
-    implementation group: 'javax.ws.rs', name: 'javax.ws.rs-api', version: '2.1.1'
     implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
+
     implementation group: 'com.amazonaws', name: 'aws-lambda-java-core', version: '1.2.0'
     implementation group: 'com.amazonaws.secretsmanager', name: 'aws-secretsmanager-caching-java', version: '1.0.1'
-    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.5.2'
-    testImplementation group: 'org.mockito', name: 'mockito-core', version: '2.1.0'
 
-    shadowJar {
-        archiveClassifier.set('')
-        zip64 true
-    }
-}
+    implementation group: 'org.zalando', name: 'problem', version: zalandoVersion
+    implementation group: 'org.zalando', name: 'jackson-datatype-problem', version: zalandoVersion
 
+    testImplementation group: 'com.github.BIBSYSDEV', name: 'nva-testutils', version: 'v0.1.6'
+    testImplementation group: 'org.hamcrest', name: 'hamcrest', version: '2.2'
+    testImplementation group: 'org.mockito', name: 'mockito-core', version: '3.2.4'
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junit5Version
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-migrationsupport', version: junit5Version
```

Comment:
What are you using the migrationsupport for?

Reply:
Will remove.